### DISCUSSION TITLE
We've been selected for the API Days Paris 2024 !!! 🔥

### DISCUSSION THREAD

**Discussion by EwenQuim** [go-fuego/fuego#227] [Announcements]
Hi contributors, 

I'm pleased to announce that we've been selected to talk about Fuego at the [API Days Paris 2024](https://www.apidays.global/paris) !!! 🎉 (search for "Fuego" in the page)

If you have any queries about the talk or if you want a free pass, don't hesitate to tell us, we got some passes left (the conference entry is 999$ usually).

We'll try to release a new version before this date.

Happy API crafting,
Ewen

**Comment by dylanhitt** [comment]
Looking forward to the talk. This is great news!
